assets/src/blocks/Spreadsheet/SpreadsheetFrontend.js

```diff
@@ -187,9 +187,15 @@ export class SpreadsheetFrontend extends Component {
                           )
                         }
                         onClick={ () => { this.onHeaderClick( index ) } }
-                        key={ index } title={ __('Sort by', 'planet4-blocks') }>
-                        { cell }
-                        <ArrowIcon />
+                        key={ index }
+                        scope='col'
+                        title={ cell }>
+                        <button
+                          className='spreadsheet-sort-button'
+                          onClick={ () => { this.onHeaderClick( index ) } }>
```

Comment:
With the current code, if I use my keyboard to tab to a table header cell and press enter, it runs `this.onHeaderClick` twice, once for the `<button>` and once for the the `<th>` after the event propagates.

If the event propagates anyway, I guess we don't need to add an event listener to the button?

Reply:
Hmmm...Thanks. Good point. I'll have to try this out.